inventory-fusion/src/main/scala/com/normation/inventory/provisioning/fusion/FusionReportUnmarshaller.scala

```diff
@@ -426,6 +419,39 @@ class FusionReportUnmarshaller(
     report.copy(node = report.node.copy(networks = nets))
   }
 
+  /*
+   * Try to normalize ARCH. We want to provide to the user something common: i686, x86_64, ppc64, etc.
+   * We are going to use https://stackoverflow.com/questions/15036909/clang-how-to-list-supported-target-architectures
+   * as reference names.
+   * Fusion provides // we change to:
+   * - amd64 => x86_64
+   * - PowerPC if OS = AIX => ppc64
+   * - PowerPC (other cases) => ppc
+   * - 32-bit / 64-bit (windows) => x86 / x86_64
+   * - things with x86_64 / x86 / i*86 in their name => x86_64 / x86 / i*86
+   * - IA-64, i[3-9]86, x86, x86_64, arm.*, other => identical [lower case]
```

Comment:
Missing in comment that if os ) AIX, and not matching anything we default to ppc64